app/controllers/branches_controller.rb

```diff
@@ -70,6 +70,15 @@ def branch_detached_destinations
   helper_method :branch_destinations
   helper_method :branch_detached_destinations
 
+  def page_title
+    if @branch
+      "Edit branch - #{@branch.title} - MoJ Forms Editor"
```

Comment:
Isn't `@branch.title` always 'Branch X`?

In which case would this be better as:
`"Edit #{@branch.title} - MoJ Forms"`